### ISSUE TITLE
Supporting Tasks ( Microsoft To-Do )

### ISSUE THREAD

**Issue by furkanarabaci** [sirredbeard/unofficial-webapp-office#11]
Since it is a web wrapper, a support for pioneer to-do app from Microsoft could be supported.

**Comment by sirredbeard** [comment]
I am open to this suggestion. It is not normally installed as part of Office though so am leaning towards it being a separate snap app. What do you think?

**Comment by furkanarabaci** [comment]
Right, I overlooked it. It is not a part in office bundle. As it is done on Windows, having them as a separate snap packages would be the wisest solution on my end, since other products like Forms and Sway exists as well. How are we going to progress on that ?

**Comment by sirredbeard** [comment]
I think Microsoft To Do would make a good standalone option.

Maybe do an unofficial-web-office-extras snap with Forms, Sway, other stuff.

**Comment by ojoanalogo** [comment]
@furkanarabaci @sirredbeard I made a fork to have To Do as a standalone option, thanks for creating such detailed instructions on how you made this webapp and the details to publish it as a snap.

https://github.com/mxarc/unofficial-webapp-todo
